packages/@o3r/core/schematics/rule-factories/generic-updates/index.ts

```diff
@@ -20,6 +23,29 @@ ${folderToExclude}
         `;
         }
       });
+      const packageManager = getPackageManager({workspaceConfig});
+      if (packageManager === 'yarn') {
+        gitignore += `
+
+### yarn ###
+# https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
+
+.yarn/*
+!.yarn/releases
+!.yarn/patches
+!.yarn/plugins
+!.yarn/sdks
+!.yarn/versions
+
+# if you are NOT using Zero-installs, then:
+# comment the following lines
+# !.yarn/cache
+
+# and uncomment the following lines
+# .pnp.*
```

Comment:
seems like one the 2 should be uncommented, no ?

Reply:
Indeed :). Updated